README.md

```diff
@@ -53,7 +53,7 @@ Command Line Utility for Gulp
       <td>Manually set the CWD. The search for the gulpfile, as well as the relativity of all requires will be from here.</td>
     </tr>
     <tr>
-      <td>--verify</td>
+      <td>--verify [path (optional)]</td>
```

Comment:
can you also update this in the cli-options.js?  I totally forgot we accepted a path